### PR TITLE
[Glimmer] Make helpers amenable to testing

### PR DESCRIPTION
**Pull request by nickiaconis**

### CODE REVIEW

File: packages/ember-views/lib/views/core_view.js [hunk 1]
```diff
@@ -7,9 +7,9 @@ import ActionHandler, { deprecateUnderscoreActions } from 'ember-runtime/mixins/
 import { typeOf } from 'ember-runtime/utils';
 
 import { InteractiveRenderer } from 'ember-htmlbars/renderer';
+import DOMHelper from 'ember-htmlbars/system/dom-helper';
```
Comment (krisselden) [inline packages/ember-views/lib/views/core_view.js]: why did you change this?  this was for a path only used by tests in ember-views we are working to get rid of.

**Comment by krisselden** [comment]
I'm not sure why we are testing helpers in ember-views, are these not covered by tests in ember-glimmer/ember-htmlbars?

File: packages/ember-application/tests/system/dependency_injection/default_resolver_test.js [hunk 2]
```diff
@@ -91,22 +88,6 @@ QUnit.test('the default resolver resolves *:main on the namespace', function() {
   detectEqual(application.FooBar, locator.lookupFactory('foo-bar:main'), 'looks up FooBar type without name on application');
 });
 
-QUnit.test('the default resolver resolves helpers', function() {
```
Comment (mixonic) [inline packages/ember-application/tests/system/dependency_injection/default_resolver_test.js]: lol

**Comment by mixonic** [comment]
looks pretty good pending some followup on @krisselden's comment.

File: packages/ember-views/lib/views/core_view.js (hunk 1 above)
Comment (nickiaconis) [inline packages/ember-views/lib/views/core_view.js]: @krisselden I was trying to make it so this could use `ember-htmlbars/system/dom-helper` or `ember-glimmer/dom` as necessary in order to make tests pass, but then I talked to @chadhietala regarding these. He said "Don’t port [views] They will be deleted", so I need to revise this.

Should the tests (outside of the ember-views package) that create a view in order to test something about helpers be updated to create a component instead?

**Comment by nickiaconis** [comment]
Updated to address krisselden's feedback and rebased with upstream changes.

File: packages/ember-glimmer/tests/utils/global-helpers-hash.js [hunk 3]
```diff
@@ -0,0 +1 @@
+../../../ember-htmlbars/lib/helpers.js
```
Comment (rwjblue) [inline packages/ember-glimmer/tests/utils/global-helpers-hash.js]: This is slightly confusing to me. Why would ember-glimmer point at ember-htmlbars helpers?

Can you help me understand?

**Comment by rwjblue** [comment]
I think this has been superseded by other PR's that have landed since this was submitted.
